Cargo.toml

```diff
@@ -60,7 +65,6 @@ default = ["std"]
 std = ["rand", "subtle/std"]
 alloc = []
 yolocrypto = ["avx2_backend"]
-bench = []
```

Comment:
If we remove the "bench" feature, will all the dev-dependencies being pulled in by criterion also be required when we run `cargo test`? Also this is going to be a lot of code to pull in, it'd be nice to have a way to turn it off?

Reply:
Yes, all the dev dependencies are pulled in for tests and benchmarks, but are excluded from library users. I would rather not have a `bench` feature -- I'm glad to be able to get rid of it.

Reply:
As long as someone is using `curve25519-dalek` as a library, they won't be using any of its dev dependencies, only its actual dependencies. So feature-gating them would only save code for people who want to make changes to `curve25519-dalek` itself, but don't want to run benchmarks. Since the dependencies are cached, this is basically a one-time savings.